dbus-tokio/src/connection.rs

```diff
@@ -69,16 +83,26 @@ pub fn new_system() -> Result<(IOResource<Connection>, Arc<Connection>), Error>
 
 #[test]
 fn method_call() {
-    use tokio::runtime::current_thread::Runtime;
+    use tokio::task;
+    use futures::future::FutureExt;
 
-    let mut rt = Runtime::new().unwrap();
+    let mut rt = tokio::runtime::Builder::new()
+        .basic_scheduler()
+        .enable_io()
+        .build()
+        .unwrap();
+
+    let local = task::LocalSet::new();
 
     let (res, conn) = new_session_local().unwrap();
-    rt.spawn(async { panic!(res.await); });
+    let spawner = async move {
+        task::spawn_local(async move { panic!(res.await);});
+    };
```

Comment:
This looks a bit superfluous? Look at the example here: https://docs.rs/tokio/0.2.4/tokio/task/struct.LocalSet.html#method.spawn_local

Any reason we can't simplify in the same way?